schema/project-level/project-schema.json

```
@@ -2468,6 +2468,16 @@
           "items": {
             "$ref": "#/definitions/Meeting"
           }
+        },
+        "landCompensationBudget": {
+          "title": "Land compensation budget",
+          "description": "The budget allocated for compensation of land.",
```

'Compensation of land' reads oddly.

```suggestion
          "description": "The budget allocated for land compensation.",
```